config/filesystems.php

```diff
@@ -41,48 +41,48 @@
 	*/
 
 	'disks' => [
-		'local' => [
-			'driver' => 'local',
-			'root' => storage_path('app'),
-		],
-
+		// Lychee uses the disk "images" to store the media files
 		'images' => [
 			'driver' => 'local',
 			'root' => env('LYCHEE_UPLOADS', public_path('uploads/')),
 			'url' => env('LYCHEE_UPLOADS_URL', 'uploads/'),
 			'visibility' => 'public',
 		],
 
+		// This is an example how the "images" disk can be hosted on an AWS S3
+		// ATTENTION: This is NOT supported yet!!!
+		// This is only a placeholder/reminder for the future
+		/*
+		'images' => [
+			'driver' => 's3',
+			'key' => env('AWS_ACCESS_KEY_ID'),
+			'secret' => env('AWS_SECRET_ACCESS_KEY'),
+			'region' => env('AWS_DEFAULT_REGION'),
+			'bucket' => env('AWS_BUCKET'),
+			'url' => env('AWS_URL'),
+			'endpoint' => env('AWS_ENDPOINT'),
+		],*/
+
+		// Lychee uses this disk to store the customized CSS file provided by the user
+		// ATTENTION: This disk MUST ALWAYS point to the local `./public/dist` directory.
+		// TODO: Maybe we should drop this Flysystem disk, because neither the driver nor the root must be changed and hence the whole point of using the Flysystem abstraction is gone.
 		'dist' => [
 			'driver' => 'local',
 			'root' => env('LYCHEE_DIST', public_path('dist/')),
 			'url' => env('LYCHEE_DIST_URL', 'dist/'),
 			'visibility' => 'public',
 		],
 
+		// Lychee uses this disk to create ephemeral, symbolic links to photos,
+		// if the feature is enabled.
```

Comment:
Presumably this is also true.
```suggestion
		// if the feature is enabled.
		// For this feature to work, the "images" disk must use the "local" driver.
```